<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgBusstopFlagEditor</class>
 <widget class="QWidget" name="WdgBusstopFlagEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>142</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbStandard">
     <property name="text">
      <string>Standard Busstop</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbFinal">
     <property name="text">
      <string>Final Busstop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbTransfer">
     <property name="text">
      <string>Transfer Busstop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbCentralTransfer">
     <property name="text">
      <string>Central Transfer Busstop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbInternal">
     <property name="text">
      <string>Internal Busstop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
